### PR TITLE
feat: support more zil eco projects

### PR DESCRIPTION
**Pull request by renlulu**

### CODE REVIEW

File: projects/equal/project.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: EQUAL
+description: Equal Wallet is an ERC20 wallet that unlocks the decentralized web through the use of their Ethereum dApp browser
+categories: false
```
Comment (AshIgnYeo) [inline projects/equal/project.md:4]: false categories

File: projects/emanthan-research/project.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Emanthan Research
+description: False
+categories: education
+status: N/A
```
Comment (AshIgnYeo) [inline projects/emanthan-research/project.md:5]: status either "live" or "building"

**Review by AshIgnYeo — CHANGES_REQUESTED** [review]

File: projects/emanthan-research/project.md (hunk 2 above)
Comment (AmritKumar) [reply projects/emanthan-research/project.md:5]:
```suggestion
status: building
```

File: projects/equal/project.md (hunk 1 above)
Comment (AmritKumar) [reply projects/equal/project.md:4]:
```suggestion
categories: wallet
```

**Review by AshIgnYeo — APPROVED** [review]

**Pull request merged**